app/controllers/ops_controller/settings/schedules.rb

```diff
@@ -489,6 +491,11 @@ def build_search_filter_from_params
       when "host" then {"=" => {"field" => "Host-name", "value" => params[:filter_value]}}
       else             {"IS NOT NULL" => {"field" => "Host-name"}}
       end
+    when "container_image"
+      case params[:filter_typ]
+      when "ems" then {"=" => {"field" => "ContainerImage.ext_management_system-name", "value" => params[:filter_value]}}
+      when "container_image" then {"=" => {"field" => "ContainerImage-name", "value" => params[:filter_value]}}
+      end
```

Comment:
@moolitayer I have no idea how you found each of these places you needed to do these `case... when "container_image"`.  I have no idea if you found them all though.  It's pretty bad we have to do these similar changes in multiple places.